modules/_labs/select/react/stories/stories.tsx

```diff
@@ -316,6 +323,7 @@ storiesOf('Labs|Select/React/Visual Testing', module)
           <SelectOption
             {...props}
             onChange={() => {}} // eslint-disable-line no-empty-function
+            style={{width: 100}}
```

Comment:
Curious what others think. 🤔 

I don't think it's wise to shorten the width here. That's against the grain of visually testing our components the way they're intended to render. This is like testing a specific style override.

I get that it's really wide but I would probably favor adding a second static state table below this one that just shows assistive focus if that's really what this line is mitigating.

Reply:
[We already do it on `TextInput`](https://github.com/Workday/canvas-kit/blob/master/modules/form-field/react/stories/stories_TextInput.tsx#L205) 🤷‍♂️ 

Not a big deal IMO.

Reply:
Hmm. That is interesting. I'm okay with it on our interactive state tables. We should have some visual baselines of the component though. Like on Buttons - small, medium, large, small-grow, medium-grow, large-grow, etc. Those would not modify things like `width`.

Reply:
Originally, I added that width to force all of the options to render at the same width since I noticed the more lengthy column headers were making some columns wider than others, which in turn made the options different widths (since they expanded to fill their parent). I thought it was strange to have some options wider than others. However, I just tried removing the `width: 100` and everything looks fine now, which makes me wonder if I had longer column headers at some point while I was finalizing the wording... so I may be able to remove that width anyway (though if we remove it here, we should also remove the `width` in the `States` story as well).

Typically, the option will render at the same width as the select button (`280px` or `100%` if `grow` is `true`). Here, it's just expanding to fill the width of the table cell. Incidentally, I'm just now wondering if it's proper to be rendering an `<li>` (the option is a styled `<li>`) without the parent `<ul>`... 🤔

Reply:
@anicholls I didn't know this was done elsewhere, I still don't think it's good practice to modify the style of something we're visually testing as our baseline. So I don't think while it may not be a big deal, I think it makes testing less rigorous. Like imagine if we wrote some styles that assumed that `width` wasn't modified to 100 but we force it to do that here?

I would rather find a way to modify the size of the container element rather than the component inside. But taking @NicholasBoll suggestion, having a visual baseline that we test is a good alternative also. It at least visually covers the default SelectOption.

Reply:
I've removed the inline styles from `Select` and `SelectOption`. I also wrapped the `SelectOption` in a `<ul>` with a width of `120px` to keep the table at a reasonable width.